src/SFA.DAS.DATA.MANAGEMENT/Tables/ASDataPL_Comt_Accounts.sql

```diff
@@ -0,0 +1,10 @@
+﻿CREATE TABLE [ASData_PL].[Comt_Accounts]
```

Comment:
do we still need this sql to create Comt_Accounts table?

Reply:
Not required, Please remove this Sarma as discussed yday.